packages/flutter_tools/test/general.shard/ios/ios_device_install_test.dart

```diff
@@ -308,6 +310,23 @@ IOSDevice setUpIOSDevice({
   );
 }
 
-class MockArtifacts extends Mock implements Artifacts {}
-class MockCache extends Mock implements Cache {}
 class MockVmService extends Mock implements VmService {}
+
+class FakeEnvironmentArtifact extends ArtifactSet {
+  FakeEnvironmentArtifact() : super(DevelopmentArtifact.iOS);
+  @override
+  Map<String, String> get environment => <String, String>{
+    'DYLD_LIBRARY_PATH': '/path/to/libs'
+  };
+
+  @override
+  Future<bool> isUpToDate() => Future<bool>.value(true);
+
+  @override
+  String get name => 'fake';
+
+  @override
+  Future<void> update(ArtifactUpdater artifactUpdater) {
```

Comment:
If you make this method `async` you don't need a `return null`

Reply:
👍